### PR TITLE
feat(ui): add support for callout in block editor

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

Part of [#13715](https://github.com/open-metadata/OpenMetadata/issues/13715)

I worked on adding support for callout in block editor.

https://github.com/open-metadata/OpenMetadata/assets/59080942/eb0aa2f4-695b-4cd9-bcb6-ed8f7d4a07e2

#
### Type of change:

- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/block-editor.less [hunk 1]
```diff
@@ -361,3 +362,49 @@
   display: flex;
   justify-content: end;
 }
+
+.om-callout-node {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  background-color: #f8f8fa;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/block-editor.less:370]: Can we use variables instead?

File: openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/block-editor.less [hunk 2]
```diff
@@ -361,3 +362,49 @@
...
+  border-left-color: @warning-color;
+}
+.om-callout-node-tip,
+.om-callout-node-info {
+  background-color: #d1e9ff;
+  border-left-color: @info-color;
+}
+
+.om-callout-node-danger {
+  background-color: #ff4c3b33;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/block-editor.less:391]: Let's avoid using direct color instead create variables for easy updates

**Review by chirag-madlani — COMMENTED** [review]
I feel we need to work on Callout Layout a bit. Let's check with Rupesh on this

**Comment by harshach** [comment]
@Sachin-chaurasiya whats remaining for this?

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya whats remaining for this?

@harshach,need to improve the design and this will be part of 1.3

**Review by chirag-madlani — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Pull request merged**